bin/region_status.rb

```diff
@@ -53,7 +53,7 @@
 java_import org.apache.hadoop.hbase.filter.FirstKeyOnlyFilter
 java_import org.apache.hadoop.hbase.util.Bytes
 java_import org.apache.hadoop.hbase.HRegionInfo
-java_import org.apache.hadoop.hbase.MetaTableAccessor
+java_import org.apache.hadoop.hbase.CatalogAccessor
```

Comment:
Have to change name of this class? Would patch be smaller if no class name change? (New name is better but high-cost changing it)